src-clean/services/api/index.js

```diff
@@ -1,17 +1,51 @@
-import axios from 'axios'
+import fetch from 'isomorphic-fetch'
+import { apiUrl } from 'config'
 
-const facade = {}
+const api = {}
+const headers = {}
```

Comment:
On server side, will not `headers` keep its state across multiple requests? If so, this would be a serious security issue.

If you already have a server side rendering project with that, please take the following steps:

1. on `server.js`, inside the express middleware, call `api.setToken('foo')` and `console.log('token set')` (only for the first request, use a sort of flag outside the middleware);
2. still inside the middleware, check the state of api `headers` (`console.log`);
3. `npm run dev`;
4. open the url on browser and check the terminal (it should display `token set` and `{ Authorization: 'Bearer foo' }`);
5. refresh the browser and check terminal again (it should **not** display `token set`);
7. If it displays `{ Authorization: 'Bearer foo' }`, this is a security problem as one user can have the token from another user on the server side.

Anyway, I can test this later today. :)